bin/check_for_long_migration_remarks.pl

```diff
@@ -0,0 +1,18 @@
+#! /usr/bin/env perl
+
+use strict;
+use warnings;
+
+my $long_remarks = 0;
+
+# loop over every line in migrations file. If line starts with `remarks:` and is over 60 chars, inc count & log error
+open(FILE, '<:encoding(UTF-8)', './resources/migrations/000_migrations.yaml') or die $!;
+while (<FILE>) {
+    $long_remarks += 1 && print "Remark on line $. is too long (${\(length)}/60): $_"
+      if m/remarks/ && s/^\s+remarks:\s'(.*)'$/$1/ && length > 60;
+}
+
+# if we've seen any remarks that are too long print message and die
+die "$long_remarks remarks are over 60 characters long, making them incompatible with MySQL 5.1." if $long_remarks;
```

Comment:
I think you might want a `\n` at the end of this string. I currently get:
```
Remark on line 3849 is too long (122/60): Whether we                                                                    should do On-Demand caching of FieldValues.
1 remarks are over 60 characters long, making them incompatible with MySQL 5.1. at ./bin/check_for_long_migration_remarks.pl line 16, <FILE> line 3861.
```
But with the newline
```
Remark on line 3849 is too long (122/60): Whether we                                                                    should do On-Demand caching of FieldValues.
1 remarks are over 60 characters long, making them incompatible with MySQL 5.1.
```
Maybe a leading `\n` would be good too so that the failing lines and the concluding line don't run together

Reply:
not a bad idea